### DISCUSSION TITLE
Doc errors?

### DISCUSSION THREAD

**Discussion by robrankin** [GoogleCloudPlatform/cloud-foundation-fabric#2679] [General]
According to the bootstrap README, the variables `iam_groups` and `group_iam` should exist:

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/master/fast/stages/0-bootstrap#iam

I believe it was originally `group_iam` (and `iam_groups` is just a typo?), but has been superceded by `iam_by_principals`?

https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/adrs/modules/20230816-iam-refactor.md

**Comment by ludoo** [comment]
Yep, care to send a quick PR to address this? Nice find!

**Reply by robrankin** [reply]
Assume its just search/replace the incorrect var names, done: https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/2680
